src/languages/_french.json

```diff
@@ -640,6 +641,10 @@
         "fragment3": " about AMMs and token liquidity. :)",
         "title": "Insufficient Liquidity :)"
       },
+      "no_quote": {
+        "title": "No quote available",
+        "text": "We couldn’t find quotes for this swap. This could be because there isn’t enough liqudity to swap or problems with how the  token is implemented."
```

Comment:
There is an additional space here: "with how the__token is implemented"

Reply:
fixed ty ser